patch/cisco-hwmon-pmbus_core-pec-support-check.patch

```diff
@@ -0,0 +1,39 @@
+From ca2339d26b3473aa4f85006c5caff8eb01fd5435 Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Wed, 26 May 2021 11:25:14 -0700
+Subject: [PATCH] hwmon: (pmbus_core) Do not enable PEC if adapter doesn't
+ support it
+
+Added code to check if i2c bus has capability to support PEC [ Packet Error Correction ] and enable it only if both controller and bus support it
```

Comment:
1.  Please wrap the lines in commit messages after 75 characters. (Linux’ script `checkpatch.pl` should tell you.)
2.  I’d write:

    > Currently, for Packet Error Correction (PEC) only the controller is checked for support. This causes problems on the …, where a … is observed, because the ….
    >
    > So, check also if I2C bus supports PEC, and only enable it if both bus and controller support it.

Reply:
Updated patch to follow linux coding guide lines. Verified by running checkpatch.pl script on patch

msiddare@6705995e2f13:/sonic/src/sonic-linux-kernel/linux-4.19.152$ ./scripts/checkpatch.pl cisco-hwmon-pmbus_core-pec-support-check.patch 
total: 0 errors, 0 warnings, 17 lines checked

cisco-hwmon-pmbus_core-pec-support-check.patch has no obvious style problems and is ready for submission.

Patch description updated as per the suggestion